app/src/main/res/values-it/strings.xml

```diff
@@ -171,216 +171,205 @@ Halten Sie sich an die Hygiene- und Verhaltensregeln."</string>
 
     <!-- Hinweis Text -->
-    <string name="hinweis_text">"Eine Person, der du begegnet bist, wurde positiv auf COVID-19 getestet."</string>
-    <string name="button_report_illness">"Erkrankung melden"</string>
-    <string name="meldungen_meldung_title">"Wichtiger Hinweis"</string>
-    <string name="meldungen_meldung_text">"Eine Person, der du begegnet bist, wurde positiv auf COVID-19 getestet. Bitte informiere dich hier über die Verhaltensempfehlungen."</string>
-    <string name="meldungen_meldung_more_button">"Mehr erfahren"</string>
-    <string name="tracing_active_title">"Tracing aktiv"</string>
-    <string name="tracing_active_text">"Begegnungen werden anonym auf deinem Gerät gespeichert."</string>
-    <string name="tracing_error_title">"Tracing nicht aktiv"</string>
-    <string name="tracing_error_text">"Aktiviere Tracing und Bluetooth, damit Begegnungen anonym aufgezeichnet werden können."</string>
+    <string name="hinweis_text">Una persona con cui hai avuto contatto è stata testata positiva al COVID-19</string>
+    <string name="button_report_illness">Condividi i sintomi</string>
+    <string name="meldungen_meldung_title">Note</string>
+    <string name="meldungen_meldung_text">Una persona con cui hai avuto contatto è stata testata positiva al COVID-19. Leggi le informazioni e le raccomandazioni qui.</string>
+    <string name="meldungen_meldung_more_button">Più informazioni</string>
+    <string name="tracing_active_title">Tracciamento attivo</string>
+    <string name="tracing_active_text">Gli incontri sono registrati in forma anonima sul tuo device.</string>
+    <string name="tracing_error_title">Il tracciamento non è attivo.</string>
+    <string name="tracing_error_text">Attivare il Bluetooth, così da permettere di registrare gli incontri in forma anonima.</string>
 
     <!-- Meldungen: zusätzliche Info Titel -->
-    <string name="meldungen_additional_info_title">"Was bedeutet das?"</string>
+    <string name="meldungen_additional_info_title">Cosa vuol dire ?</string>
 
     <!-- Meldungen: Info Text 1 -->
-    <string name="meldungen_additional_info_text1">"Um die Verbreitung eines Virus zu stoppen, muss man die Ansteckungsketten unterbrechen. Das gelingt, wenn man ansteckende Personen isoliert. Nur: Beim neuen Coronavirus gibt es ein Problem. Es kann sein, dass du bereits ansteckend bist, bevor du dich krank fühlst. So kann sich das Virus ausbreiten, obwohl sich Personen mit Symptomen selbst isolieren. Denn wir sind einen Schritt zu spät."</string>
+    <string name="meldungen_additional_info_text1">Per bloccare il virus bisogna interrompere la catena di trasmissione. Questo si può fare isolando le persone contagiose. Con il COVID-19 le persone possono infette possono essere contagiose prima di mostrare i sintomi, il che sigifica che possono trasmettere il virus senza saperlo. Se solo le persone con sintomi vengono isolate il virus continuerà a trasmettersi. Dobbiame stare un passo avanti !</string>
```

Comment:
Per bloccare il virus bisogna interrompere la catena di trasmissione. Questo si può fare isolando le persone contaggiate. Con il COVID-19 le persone infette possono trasmettere il virus prima di mostrare i sintomi, il che significa che possono infettare senza saperlo. Questo permette al virus di propagarsi anche se le persone con i sintomi si isolano. Dobbiamo stare un passo avanti !

Reply:
Small typo in Corrado's fix

Per bloccare il virus bisogna interrompere la catena di trasmissione. Questo si può fare isolando le persone contagiate. Con il COVID-19 le persone infette possono trasmettere il virus prima di mostrare i sintomi, il che significa che possono infettare senza saperlo. Questo permette al virus di propagarsi anche se le persone con i sintomi si isolano. Dobbiamo stare un passo avanti !